Library/Formula/tarsnap.rb

```diff
@@ -11,14 +11,32 @@ class Tarsnap < Formula
     sha1 "a84965928a810a18f8dbac38091e4ab9a9e69214" => :lion
   end
 
-  depends_on "xz" => :optional
+  head do
+    url "https://github.com/Tarsnap/tarsnap.git"
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+  end
+
   depends_on "openssl"
+  depends_on "xz" => :optional
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--enable-sse2",
-                          "--prefix=#{prefix}",
-                          "--sysconfdir=#{etc}"
+    system "autoreconf", "-iv" if build.head?
+
+    args = %w[
```

Comment:
Need a big `W` rather than a little one here. The little one ignores the variables tucked inside the `#{}` brackets.